### PR TITLE
Update README.md - Add Missing Project description

### PR DESCRIPTION
**Pull request by sainthiago**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
 # DeFi Swap AI Assistant 
-
 <img src="https://i.imgur.com/xUkwJiK.png" alt="cover_image"  width="0" />
 
+Ref Finance Agent is a Next.js 14 template for creating decentralized finance (DeFi) applications.
```
Comment (Markeljan) [inline README.md]: Suggestion:

Ref Finance Agent is a template for creating a Bitte.ai Plugin for facilitating DeFi swaps using Ref Finance.  Built using Next.js 14 + Elysia.

Comment (Markeljan) [reply README.md]: Maybe also include a link to  https://docs.mintbase.xyz/ai/assistant-plugins

Comment (sainthiago) [inline README.md]:
```suggestion
Ref Finance Agent is a template for creating a Bitte.ai Plugin for facilitating DeFi swaps using Ref Finance. Built using Next.js 14 + Elysia.

[Build your own agent](https://docs.mintbase.xyz/ai/assistant-plugins)
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,15 @@
 # DeFi Swap AI Assistant 
-
 <img src="https://i.imgur.com/xUkwJiK.png" alt="cover_image"  width="0" />
 
+Ref Finance Agent is a Next.js 14 template for creating decentralized finance (DeFi) applications. 
+Built by Bitte, it facilitates the development of AI-powered DeFi swap agents. The template supports creating, managing, and deploying DeFi swap functionalities.
```
Comment (sainthiago) [inline README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (sainthiago) [reply README.md]: thanks for the suggestion :)
updated

**Review by Markeljan — APPROVED** [review]

**Pull request merged**